views/pages/urls/url.hbs

```diff
@@ -34,4 +56,33 @@
     </div>
   </div>
   <div class="col"></div>
-</div>
\ No newline at end of file
+</div>
+
+<script>
+  // Show an element
+  function show (elem) {
```

Comment:
since we are already using jQuery, just use jQuery's `hide()` and `show()` handlers